Formula/dosbox-staging.rb

```diff
@@ -25,15 +26,19 @@ def install
     args = %W[
       --prefix=#{prefix}
       --disable-dependency-tracking
-      --disable-fluidsynth
       --disable-sdltest
       --enable-core-inline
     ]
 
+    ENV.append_to_cflags "-DNDEBUG"
+    ENV.append_to_cflags "-O3"
+    ENV.append_to_cflags "-fno-math-errno"
+    ENV.append_to_cflags "-fno-strict-aliasing"
     system "./autogen.sh"
     system "./configure", *args
-    system "make", "install"
+    system "make", "CXXFLAGS=#{ENV.cflags}", "install"
```

Comment:
Are you sure about passing CFLAGS as CXXFLAGS and that make doesn't pick it up from env?

Reply:
Yes, the upstream build specifically uses the same flags for CFLAGS and CXXFLAGS; this seemed like the most parsimonious approach for the recipe.